Unable to use when path have spaces

If you are running this from a path with spaces, it fails

Steps:
1. `mkdir -p "/tmp/minefield/my project"`
2. `cd "/tmp/minefield/my project"`
2. `npm i find-chrome-driver-version`
3. `echo "console.log(require('find-chrome-driver-version').findChromeDriverVersionSync())" > test.js`
4. `node test.js`

The output:
```
internal/modules/cjs/loader.js:818
  throw err;
  ^

Error: Cannot find module '/private/tmp/minefield/my'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
child_process.js:674
    throw err;
    ^

Error: Command failed: node /private/tmp/minefield/my project/node_modules/find-chrome-driver-version/src/spawn.js
internal/modules/cjs/loader.js:818
  throw err;
  ^

Error: Cannot find module '/private/tmp/minefield/my'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

    at checkExecSyncError (child_process.js:635:11)
    at execSync (child_process.js:671:15)
    at /private/tmp/minefield/my project/node_modules/find-chrome-driver-version/src/chrome-version.js:42:13
    at /private/tmp/minefield/my project/node_modules/find-chrome-driver-version/src/util.js:28:43
    at Object.run (/private/tmp/minefield/my project/node_modules/crocks/core/compose.js:8:14)
    at Object.<anonymous> (/private/tmp/minefield/my project/node_modules/ramda/src/invoker.js:59:29)
    at Object.f1 (/private/tmp/minefield/my project/node_modules/ramda/src/internal/_curry1.js:19:17)
    at Object.<anonymous> (/private/tmp/minefield/my project/node_modules/ramda/src/internal/_pipe.js:3:14)
    at Object.findChromeDriverVersionSync (/private/tmp/minefield/my project/node_modules/ramda/src/internal/_arity.js:6:19)
    at Object.<anonymous> (/private/tmp/minefield/my project/test.js:1:51) {
  status: 1,
```

If you run the same code on path without spaces, it works.

--- a/src/chrome-version.js
+++ b/src/chrome-version.js
@@ -12,7 +12,7 @@ const { join } = require('path');
 const { execSync } = require('child_process');
 
 const CHROME_STORAGE_API = 'https://chromedriver.storage.googleapis.com/';
-const NODE_CALL_FOR_SPAWN = `node ${join(__dirname, './spawn.js')}`;
+const NODE_CALL_FOR_SPAWN = `node "${join(__dirname, './spawn.js')}"`;
 const CACHE_FILE_PATH = join(__dirname, '../.cache');
 const PROXY = process.env.ALL_PROXY || process.env.HTTPS_PROXY || process.env.HTTP_PROXY;
 

